fhd_output/fhd_quickview.pro

```diff
@@ -276,13 +276,14 @@ ENDFOR
 IF Keyword_Set(cal_source_flag) THEN cal_sources = source_image_generate(cal_source_arr_out,obs_out,pol_i=4,resolution=16,$
     dimension=dimension,restored_beam_width=restored_beam_width,_Extra=extra)
 
-; renormalize based on weights
-renorm_factor = get_image_renormalization(obs_out,weights_arr=weights_arr,beam_base=beam_base_out,filter_arr=filter_arr,$
-  image_filter_fn=image_filter_fn,pad_uv_image=pad_uv_image,degpix=degpix,/antialias)
-for pol_i=0,n_pol-1 do begin
-  *instr_dirty_arr[pol_i]*=renorm_factor[pol_i]
-  IF model_flag THEN *instr_model_arr[pol_i]*=renorm_factor[pol_i]
-endfor
+IF keyword_set(image_filter_fn) THEN IF image_filter_fn NE 'filter_uv_weighted' THEN BEGIN ; renormalize based on weights
+    renorm_factor = get_image_renormalization(obs_out,weights_arr=weights_arr,beam_base=beam_base_out,filter_arr=filter_arr,$
+      image_filter_fn=image_filter_fn,pad_uv_image=pad_uv_image,degpix=degpix,/antialias)
+    for pol_i=0,n_pol-1 do begin
+      *instr_dirty_arr[pol_i]*=renorm_factor[pol_i]/(degpix*!DtoR)^2.
+      IF model_flag THEN *instr_model_arr[pol_i]*=renorm_factor[pol_i]/(degpix*!DtoR)^2.
```

Comment:
I'm surprised I'm only catching this now, but shouldn't we also renormalize `instr_sources` if `source_flag` is set, and `instr_rings` if `ring_radius` is also set?

Reply:
You mean to get consistent scaling for the restored images? I agree in that case. This should be fixed.

Reply:
This has been fixed and tested and is ready for a code review.